bpf/bpf_overlay.c

```diff
@@ -355,6 +358,9 @@ static __always_inline int handle_ipv4(struct __ctx_buff *ctx,
 		/* See comment at equivalent code in handle_ipv6() */
 		if (info && identity_is_remote_node(*identity))
 			*identity = info->sec_identity;
+
+		if (info && identity_is_world_ipv4(*identity))
+			*identity = info->sec_identity;
```

Comment:
Nit: How about doing like this?

```
		if (info && (identity_is_remote_node(*identity) || identity_is_world_ipv4(*identity)))
			*identity = info->sec_identity;
```

Reply:
Thank you! I've fixed :) 
Also, I added some comments just before this block.